header:
  stamp:
    sec: 1707378814
    nanosec: 324426758
  frame_id: camera_depth_optical_frame
height: 720
width: 1280
distortion_model: plumb_bob
d:
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
k:
- 636.8314208984375
- 0.0
- 643.933837890625
- 0.0
- 636.8314208984375
- 359.81732177734375
- 0.0
- 0.0
- 1.0
r:
- 1.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 1.0
p:
- 636.8314208984375
- 0.0
- 643.933837890625
- 0.0
- 0.0
- 636.8314208984375
- 359.81732177734375
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
binning_x: 0
binning_y: 0
roi:
  x_offset: 0
  y_offset: 0
  height: 0
  width: 0
  do_rectify: false
